azure-pipelines.release.yml

```diff
@@ -0,0 +1,64 @@
+trigger:
+  branches:
+    include:
+      - release-*
+
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: AzurePipelines-EO
+      image: 1ESPT-Ubuntu22.04
+      os: linux
+
+    stages:
```

Comment:
Normally I would not use stages, but since this is a template, all of this stuff are parameters specific to the template, so I have to use them.